### DISCUSSION TITLE
Question about accesing nocodb behind nginx proxy manager

### DISCUSSION THREAD

**Discussion by vuebitsdev** [nocodb/nocodb#9289] [Q&A]
Hi i'm trying to configure nocodb with a separate Nginx Proxy Manager, running in other container. I added the network of NPM to the installation command of nocodb. I can access nocodb using the ip and port 8081, but when i try to access with my domain i always get error 502. In NPM i set a proxy  to http://nocodb:8081 i also tried  http://127.0.0.1:8081 and http://localhost:8081

I got the same error with Openproject but that was resolved by just starting the container in the same network as NPM and making a proxy to http://openproject:8080.

My settings in NPM
<img width="482" alt="image" src="https://github.com/user-attachments/assets/a9f30fd1-7f95-4de0-9261-d1b059e31236">

The command that i use to run nocodb is. I changed the port to 8081 because the container of openproject is using that port:

`docker run -d -p 8081:8080 --name nocodb \
--restart=always \
--network=same_npm_network \
-v "$(pwd)"/nocodb:/usr/local/docker/my-nocodb/data/ \
-e NC_PUBLIC_URL="https://nocodb.mydomain.com" \
-e NUXT_PUBLIC_NC_BACKEND_URL="https://nocodb.mydomain.com" \
-e NC_SMTP_FROM=nocodb@mydomain.com \
-e NC_SMTP_HOST=smtp.myemail.com \
-e NC_SMTP_PORT=587 \
-e NC_SMTP_USERNAME=my@smtpuser.com \
-e NC_SMTP_PASSWORD=my_smtp_pass \
-e NC_SMTP_SECURE=true \
-e NC_ADMIN_EMAIL=admin.nocodb@mydomain.com \
-e NC_ADMIN_PASSWORD=admin_pass \
-e NC_INVITE_ONLY_SIGNUP=true \
nocodb/nocodb:latest`

I'm doing something wrong? thank you for your help!.

**Comment by ubiquities** [comment]
Hopefully you already got this figured out but for anyone else that finds this post. Remember to check your firewall on the host machine. 

I had this issue, and it turned out to be the simplest thing. I setup the port to be 8181, and forgot to update `sudo ufw allow 8181`
I kept a sticky note on my monitor reminding me to check ufw, which helped a lot, figured I finally learned my lesson and removed the note and immediately started forgetting about opening the necessary ports.
